docs/source/elastic/agent.rst

```diff
@@ -59,3 +59,18 @@ to implement.
    :private-members:
 
 .. autoclass:: torch.distributed.elastic.agent.server.api.RunResult
+
+
+Watchdog in the Agent
```

Comment:
We should document this on the local agent class and then can have this section at the expiration timers page